src/shelly_switch.cpp

```diff
@@ -25,6 +25,14 @@
 
 namespace shelly {
 
+const HAPUUID kHAPCharacteristic_EveConsumption = {
```

Comment:
please add comments on what these are.

Reply:
and make both static.

Reply:
What kind of comment do you mean? For me the name says enough, please make a suggestion.

About making them static, I don't know how, `static const` wouldn't compile.

Note: https://github.com/mongoose-os-apps/shelly-homekit/blob/master/src/shelly_hap_doorbell.cpp#L25 uses the same notation as here, no static, no comment.

Reply:
magic numbers need comments, there are a lot of magic numbers here, we need a link to where they come from, to give context.
the doorbell thing could use a comment too - saying standalone doorbell service is deprecated, etc.